modules/op-installing-tkn-on-linux-using-rpm.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/tkn_cli/installing-tkn.adoc
+
+[id="installing-tkn-on-linux-using-rpm"]
+
+= Installing {pipelines-title} CLI (tkn) on Linux using an RPM
+
+For {op-system-base-full}, you can install the {pipelines-title} CLI (`tkn`) as an RPM if you have an active {product-title} subscription on your Red Hat account.
+
+.Prerequisites
+
+* Must have root or sudo privileges.
+
+.Procedure
+
+. Register with Red Hat Subscription Manager:
++
+----
+# subscription-manager register
+----
+
+. Pull the latest subscription data:
++
+----
+# subscription-manager refresh
+----
+
+. List the available subscriptions:
++
+----
+# subscription-manager list --available --matches '*OpenShift*'
```

Comment:
subscription-manager list --available --matches '*pipelines*'

Reply:
I thought subscription would be for the OCP, and not specifically for Pipelines. The same was mentioned for Serverless - https://docs.openshift.com/container-platform/4.4/serverless/installing_serverless/installing-kn.html#installing-cli-linux-rpm_installing-kn

Reply:
I faced problem while trying to install `kn` using the steps mentioned in the link above. I used `subscription-manager list --available --matches 'serverless'` and it worked well for me :)

Reply:
@praveen4g0  @VeereshAradhya   I have made the changes, pls verify.